pages/linux/zforce.md

```diff
@@ -0,0 +1,8 @@
+# zforce
+
+> Add a '.gz' extension on all supplied gzip files.
```

Comment:
```suggestion
> Add a GZ extension on all supplied GZip files.
```
For page descriptions, we could go with capitalized extensions/names. (We can leave the examples using quotes to wrap extensions as it is)